### PR TITLE
mohammad/empty_table_msg

### PR DESCRIPTION
**Pull request by binary-hamedanchi**

### CODE REVIEW

File: src/javascript/binary/static/profittable/profittable.init.js [hunk 1]
```diff
@@ -45,6 +45,19 @@ var ProfitTableWS = (function () {
...
             ProfitTableUI.createEmptyTable().appendTo("#profit-table-ws-container");
             ProfitTableUI.updateProfitTable(getNextChunk());
+
+            // Show a message when the table is empty
+            if(transactionsReceived === 0) {
+                $('#profit-table tbody')
+                    .empty()
+                    .append($('<tr/>', {class: "flex-tr"})
+                        .append($('<td/>', {colspan: 7})
+                            .append($('<p/>', {class: "notice-msg center", text: text.localize("Your first trade will appear here, once you've purchased it.")})
```
Comment (jy-binary) [inline src/javascript/binary/static/profittable/profittable.init.js]: Please change the message to "Your account has no trading activity."

File: src/javascript/binary/static/statement/statement.init.js [hunk 2]
```diff
@@ -36,6 +36,19 @@ var StatementWS = (function(){
...
             StatementUI.createEmptyStatementTable().appendTo("#statement-ws-container");
             StatementUI.updateStatementTable(getNextChunkStatement());
+
+            // Show a message when the table is empty
+            if(transactionsReceived === 0) {
+                $('#statement-table tbody')
+                    .empty()
+                    .append($('<tr/>', {class: "flex-tr"})
+                        .append($('<td/>', {colspan: 6})
+                            .append($('<p/>', {class: "notice-msg center", text: text.localize("Your first trade will appear here, once you've purchased it.")})
```
Comment (jy-binary) [inline src/javascript/binary/static/statement/statement.init.js]: Please change the message to "Your account has no trading activity."

**Comment by jy-binary** [comment]
Can you deploy this to a QA box?

File: src/javascript/binary/static/profittable/profittable.init.js (hunk 1 above)
Comment (binary-hamedanchi) [inline src/javascript/binary/static/profittable/profittable.init.js]: Message changed.

File: src/javascript/binary/static/statement/statement.init.js (hunk 2 above)
Comment (binary-hamedanchi) [inline src/javascript/binary/static/statement/statement.init.js]: Message changed.

**Comment by binary-hamedanchi** [comment]
It's live on QA34:
- https://www.binaryqa34.com/user/statementws?l=EN
- https://www.binaryqa34.com/user/profit_table?l=EN

**Pull request merged**